packages/manifest/src/manifest.ts

```diff
@@ -545,7 +545,7 @@ export class DevPreviewManifest implements DevPreviewSchema {
   localizationInfo?: ILocalizationInfo;
 
   developer: IDeveloper = {
-    name: "Teams App, Inc.",
+    name: "Contoso",
```

Comment:
I think it's better to change the name in devPreviewManifest.ts, otherwise teams app's manifest is also changed.

Reply:
Isn't devPreviewManifest.ts just the definition of the interfaces representing the dev preview manifest structure?  This is where those interfaces become a class with values.  The DevPreviewManifest object gets this change which isn't specific to the type of app getting created while the TeamsAppManifest class keeps the Teams specific name.  

When we eventually unify these two classes (or deprecate one) we're going to have to have a generic value the represents the whole rather than being teams specific.

Reply:
Yeah, you are right.